editor/editor_dock_manager.cpp

```diff
@@ -327,6 +362,44 @@ void EditorDockManager::_dock_make_selected_float() {
 	_edit_current();
 }
 
+void EditorDockManager::bottom_dock_show_placement_popup(const Rect2i &p_position) {
+	dock_bottom_selected_idx = bottom_docks.find(FileSystemDock::get_singleton());
```

Comment:
That's still hard-coded FileSystem dock 🙃
You could take the dock as argument, which would allow every dock to handle itself.

Reply:
ah shit did not pay attention to this. heh.